### ISSUE TITLE
Building paperback is slow

### ISSUE THREAD

**Issue by Sjors** [Sjors/nado-book#50]
`./make_book.sh --paperback` takes really long for me.

**Comment by bradleystachurski** [comment]
Out of curiosity, is this roughly what you're seeing for build times?
```bash
root@eaeb68a17691:~/nado-book# time ./make_book.sh --paperback

real    3m4.038s
user    2m59.142s
sys     0m13.953s
```

For reference, I'm not on a particularly beefy machine.
```bash
root@eaeb68a17691:~/nado-book# lscpu
Architecture:        x86_64
CPU(s):              4
Model name:          Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz
```

**Comment by Sjors** [comment]
Yes, 3 to 7 minutes depending on the machine. Added some stats to the description.

It's especially annoying when I'm making smaller changes. Perhaps it can be parallelised, so that it only rebuilds the chapter I touched? It might also help to replace the current bash script with a makefile, which should avoid some duplicate work on the figures.
